lib/email/message_builder.rb

```diff
@@ -126,6 +126,7 @@ def subject
       else
         subject = @opts[:subject]
       end
+      subject = DiscoursePluginRegistry.apply_modifier(:user_notification_subject, subject,@opts)
```

Comment:
Message builder is also used for other e-mails type, not only notifications. 

The modifiers name should be more generic, something like `message_builder_subject` for example.